mypyc/irbuild/mapper.py

```diff
@@ -73,6 +73,11 @@ def type_to_rtype(self, typ: Type | None) -> RType:
 
         typ = get_proper_type(typ)
         if isinstance(typ, Instance):
+            if typ.type.is_newtype and len(typ.type.bases) == 1:
```

Comment:
It is better to assert the `len() == 1` instead of `if`. If we have a NewType with multiple bases, something is seriously broken.